$NetBSD: patch-util_Makefile,v 1.1 2023/01/22 17:41:52 vins Exp $

Fix the invalid DESTDIR path containing '/usr' and use the
pkgsrc macros.

--- util/Makefile.orig	2020-08-09 17:20:37.000000000 +0000
+++ util/Makefile
@@ -29,21 +29,18 @@ endif
 install-common:
 	@if [ -x pdfconcat ] ; then \
 	mkdir -p "$(DESTDIR)"/usr/bin ; \
-	install -c pdfconcat $(DESTDIR)/usr/bin ; \
-	echo "install -c pdfconcat $(DESTDIR)/usr/bin" ; \
-	mkdir -p "$(DESTDIR)"/usr/share/man/man1 ; \
-	install -c -m 644 pdfconcat.1 "$(DESTDIR)"/usr/share/man/man1 ; \
-	echo "install -c pdfconcat.1 "$(DESTDIR)"/usr/share/man/man1" ; fi
+	${BSD_INSTALL_PROGRAM_DIR} ${DESTDIR}${PREFIX}/bin ; \
+	${BSD_INSTALL_PROGRAM} pdfconcat ${DESTDIR}${PREFIX}/bin ; \
+	${BSD_INSTALL_MAN_DIR} ${DESTDIR}${MANDIR}/man1 ; \
+	${BSD_INSTALL_MAN} pdfconcat.1 ${DESTDIR}${MANDIR}/man1 ; fi
 	@if [ -x ppmtops ] ; then \
-	mkdir -p "$(DESTDIR)"/usr/bin ; \
-	install -c ppmtops "$(DESTDIR)"/usr/bin ; \
-	echo "install -c ppmtops "$(DESTDIR)"/usr/bin" ; fi
+	${BSD_INSTALL_PROGRAM_DIR} ${DESTDIR}${PREFIX}/bin ; \
+	${BSD_INSTALL_PROGRAM} ppmtops ${DESTDIR}${PREFIX}/bin ; fi
 
 install-pgf:
 	@if [ -x pgf2pnm ] ; then \
-	mkdir -p "$(DESTDIR)"/usr/bin ; \
-	install -c pgf2pnm "$(DESTDIR)"/usr/bin ; \
-	echo "install -c pgf2pnm "$(DESTDIR)"/usr/bin" ; fi
+	${BSD_INSTALL_PROGRAM_DIR} ${DESTDIR}${PREFIX}/bin ; \
+	${BSD_INSTALL_PROGRAM} pgf2pnm ${DESTDIR}${PREFIX}/bin ; fi
 
 ifneq ($(WITH_PGF),no)
 install: install-common install-pgf
